apps/src/templates/sectionProgress/progressTables/ProgressTableStudentName.jsx

```diff
@@ -92,16 +92,17 @@ export default class ProgressTableStudentName extends React.PureComponent {
         aria-describedby={tooltipId}
       >
         {this.renderTooltip()}
-        {studentUrl && (
```

Comment:
nice cleanup here! (i didn't notice that sloppy conditional logic when i copied this from the legacy component). 

however, there is no valid scenario in which `scriptUrlForStudent` returns `null`, so i propose we clean this up even more and remove the conditional entirely. here's my proposal:
1) add a `name: PropTypes.string.isRequired` to `sectionProgressConstants.scriptDataPropType`
2) add `studentUrl: PropTypes.string.isRequired` to `ProgressTableStudentName.propTypes` and remove `scriptName`
3) call `scriptUrlForStudent` in `ProgressTableStudentList` and pass it into `ProgressTableStudentName` (since the `scriptDataPropType` is enforced in `ProgressTableStudentList`
4) remove this conditional and always render the link.

Reply:
ahh, so much cleaner! :)